import-scripts/merge_clinical_metadata_headers.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import sys
+#import optparse
+#import os
+#import shutil
+#import re
+#import csv
+
```

Comment:
Please put a comment here describing the args

Reply:
comments added (for script args/use and per method)

Reply:
comments added (for script args/use and per method)